docs/rotate-oidc-key.md

```diff
@@ -11,17 +11,17 @@ When OpenShift is configured to use temporary credentials (AZWI, STS, WIF) to au
 
     Common
     ```bash
-    CLUSTER_ID=$(oc get infrastructures cluster -o jsonpath='{.status.infrastructureName}')
-
-    CLUSTER_NAME=${CLUSTER_ID:0:-6}
-
-    TEMPDIR=${CLUSTER_NAME}-$(date +%Y%m%d%H%M%S)
+    TEMPDIR=$(mktemp -d)
 
     mkdir ${TEMPDIR}
```

Comment:
I think this should be removed since `mktemp -d` has created a dir already?

Reply:
Good catch, I was thinking in circles. [Done](https://github.com/openshift/cloud-credential-operator/compare/ad0444b50a9a4eab293fadc4f3d74a8bee5c5590..4a81f907023b9fb6d2b579d2b542a5d5a8b9cd00).